src/JavaDependenciesManager.ts

```diff
@@ -19,69 +19,66 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 
 const PREFERENCE_KEY_JAVA_REFERENCED_LIBRARIES = "java.project.referencedLibraries";
+export const CAMEL_VERSION = "3.3.0";
 
-export var areJavaDependenciesDownloaded = false;
```

Comment:
this flag is not used anymore. We are using an indirect check, looking at the filesystem. With VS Code 1.46.0, the field was always seen as undefined.